.github/workflows/alerts.yaml

```
@@ -15,10 +15,19 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Render and test lifecycle 
-        uses: scality/action-prom-render-test@1.0.1
+        uses: scality/action-prom-render-test@1.0.2
         with:
           alert_file_path: monitoring/lifecycle/alerts.yaml
           test_file_path: monitoring/lifecycle/alerts.test.yaml
           alert_inputs: >-
             namespace=zenko,job_lifecycle_producer=artesca-data-backbeat-lifecycle-producer-headless,job_lifecycle_bucket_processor=artesca-data-backbeat-lifecycle-bucket-processor-headless,job_lifecycle_object_processor=artesca-data-backbeat-lifecycle-object-processor-headless,lifecycle_conductor_replicas=1,lifecycle_bucket_replicas=3,lifecycle_object_replicas=3
           github_token: ${{ secrets.GIT_ACCESS_TOKEN }}
+
+      - name: Render and test replication 
+        uses: scality/action-prom-render-test@1.0.2
```

nit: Usually we prefer have a "moving" tag v1 on the action that is retagged on each new 1.x version so that workflows can directly inherit from the latest action changes.

```suggestion
        uses: scality/action-prom-render-test@v1
```